packages/openhim-core-api/src/jembi-openhim-core-api.ts

```diff
@@ -5,7 +5,7 @@ if running locally set REACT_APP_OPENHIM_API_BASE_URL environment variable to po
  **************************************************************************************************************** */
 
 const API_URL =
-  process.env.REACT_APP_OPENHIM_API_BASE_URL || 'https://localhost:8080/'
+  process.env.REACT_APP_OPENHIM_API_BASE_URL || 'https://13.245.11.170:8080/'
```

Comment:
Should this not be reverted for local dev? If we want to provide QA details we should use the env var.

Reply:
This has been reverted and added to the build script